chrome-extension/src/App.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import './App.css';
+import {v4 as uuid} from "uuid";
+
+function App() {  
+
+  const [userName,setUserName]=useState()
+  const [userEmail,setUserEmail]=useState("")
+  const [isEmail,setIsEmail]=useState(false)
+  const [showName,setShowName]=useState(true)
```

Comment:
you can reduce the number of usestate by using object

Reply:
ok